notify/mail.sh

```diff
@@ -62,7 +62,7 @@ mail_send() {
   fi
 
   contenttype="text/plain; charset=utf-8"
-  subject="=?UTF-8?B?$(echo "$_subject" | _base64)?="
+  subject="=?UTF-8?B?$(echo -n "$_subject" | _base64)?="
```

Comment:
don't use  `echo -n` option, it's not supported on all the platforms.
please use  `printf "%b"` instead.